src/components/TodoList/TodoList.tsx

```diff
@@ -1,210 +1,82 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useAppSelector } from '../../app/hooks';
+import { TodoInfo } from '../TodoInfo/TodoInfo';
+
+function isContains(string: string, substring: string): boolean {
+  return string.toLocaleLowerCase().includes(substring.toLocaleLowerCase());
+}
 
 export const TodoList: React.FC = () => {
+  const todos = useAppSelector(state => state.todos);
+  const filterParams = useAppSelector(state => state.filter);
+  const [visibleTodos, setVisibleTodos] = useState(todos);
+
+  const getVisibleTodos = () => {
```

Comment:
```suggestion
  const updateVisibleTodos = () => {
```
функції називаємо get... коли вони мають щось повертати